src/services/recovery/recovery-state.ts

```diff
@@ -48,7 +41,7 @@ export const _getSafeCreationReceipt = memoize(
     safeAddress: string
     provider: JsonRpcProvider
   }): Promise<TransactionReceipt> => {
-    const url = `${transactionService}/v1/${safeAddress}/creation/`
+    const url = `${transactionService}api/v1/safes/${safeAddress}/creation/`
```

Comment:
It seems as though [there can be a forward slash on some networks](https://safe-client.safe.global/v1/chains). Perhaps we should check if one is required and/or fix the configs.